inc/sender_test_case.json

```diff
@@ -14,9 +15,12 @@
                     "Temp_Type": "Celcius"
                 }
             ],
-            "Temperature_Check_Attemp": {"Stream Length": 50, "Required": true},
-            "SOC_Check_Attemp": {"Stream Length": 50, "Required": false},
```

Comment:
code is not simulating other parameters except temperature. 
Ran code with "{"Name": "SOC", "Required": true},"

Reply:
SOC is now printed

Reply:
Thank you. Now I can see from Actions output.